include/config/config_audio.h

```diff
@@ -38,6 +38,6 @@
 
 /** 
  * Uses a much better implementation of reverb over vanilla's fake echo reverb. Great for caves or eerie levels, as well as just a better audio experience in general.
- * Reverb parameters can be configured in audio/synthesis.c to meet desired aesthetic/performance needs. Currently US/JP only. Hurts emulator and console performance.
+ * Reverb presets can be configured in audio/data.c to meet desired aesthetic/performance needs. Currently US/JP only. May hurt console and emulator performance if used carelessly.
```

Comment:
> if used carelessly
i think you should specify why or what careless means, this comment is somewhat terrifying lol

Reply:
Idk how to make this more concise lol, I don't want to write an essay here when the Wiki page and the `data.c` descriptions exist to do that part for me. I guess I'm equally as afraid of making it sound like you should just never use it because performance = bad.

Reply:
Actually I think I'm just going to remove the last two sentences completely. US/JP isn't exactly relevant since EU and SH aren't even usable anymore, plus performance is already implied by the sentence before it.